CMakeLists.txt

```diff
@@ -206,6 +206,10 @@ if(${VELOX_FORCE_COLORED_OUTPUT})
   elseif("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang"
          OR "${CMAKE_CXX_COMPILER_ID}" STREQUAL "AppleClang")
     add_compile_options(-fcolor-diagnostics)
+    if(CMAKE_SYSTEM_NAME STREQUAL "Linux" AND "${CMAKE_CXX_COMPILER_VERSION}"
```

Comment:
Nit: Its a bit wierd that this is gated at the top level with the FORCE_COLORED_OUTPUT flag. If I turn that off I cant compile with clang15 ?